evap/settings.py

```diff
@@ -420,6 +420,10 @@ def CHARACTER_ALLOWED_IN_NAME(character):  # pylint: disable=invalid-name
 
 # speed up tests
```

Comment:
This comment seems out of place now that more stuff is going on in the branch below

Reply:
altered the comment